switchapi/es2k/switch_fdb.c

```diff
@@ -291,9 +291,19 @@ switch_status_t switch_api_l2_forward_create(const switch_device_t device,
 
       status = switch_rif_get(device, api_l2_info->rif_handle, &rif_info);
       CHECK_RET(status != SWITCH_STATUS_SUCCESS, status);
+
+      /* Get appropiate port_id if not already set */
+      switch_pd_to_get_port_id(&rif_info->api_rif_info);
+
+      /* Get physical port ID from its corresponding PR VSI ID */
       status = switch_pd_get_physical_port_id(
           device, rif_info->api_rif_info.port_id, &api_l2_info->port_id);
       CHECK_RET(status != SWITCH_STATUS_SUCCESS, status);
+
+      /* Get bridge ID from its physical port ID */
+      status = switch_pd_get_bridge_id(device, api_l2_info->port_id,
+                                       &api_l2_info->bridge_id);
+      CHECK_RET(status != SWITCH_STATUS_SUCCESS, status);
```

Comment:
What happens if the port is not part of any bridge?

Reply:
It is an error case, we always need to associate a port to a bridge in LNW-v2